### ISSUE TITLE
Marvin issues multiple warnings in PY3

### ISSUE THREAD

**Issue by albireox** [sdss/marvin#159]
I haven't tracked the origin, but this happens every time I run anything (even importing a simple Map) in PY3

```
INFO: No release version set. Setting default to MPL-5
WARNING: can't resolve package from __spec__ or __package__, falling back on __name__ and
         __path__
WARNING: can't resolve package from __spec__ or __package__, falling back on __name__ and
         __path__
WARNING: unclosed file <_io.TextIOWrapper
         name='/Users/albireo/Documents/Repo/manga/marvin/python/marvin/db/dbcon
         fig.ini' mode='r' encoding='UTF-8'>
WARNING: unclosed file <_io.TextIOWrapper
         name='/usr/local/lib/python3.6/site-packages/astropy/extern/bundled/six
         .py' mode='r' encoding='utf-8'>
WARNING: Not importing directory /usr/local/lib/python3.6/site-packages/mpl_toolkits:
         missing __init__
WARNING: can't resolve package from __spec__ or __package__, falling back on __name__ and
         __path__
WARNING: Flags not at the start of the expression <(.+?)(\s.*?)?>.*?</ (truncated)
WARNING: can't resolve package from __spec__ or __package__, falling back on __name__ and
         __path__
WARNING: object of type <class 'float'> cannot be safely interpreted as an integer.
```

I'm assigning it to the 2.1 milestone because I think we should try to clean the output before we freeze marvin.

**Comment by albireox** [comment]
This has gotten better in the last few versions since we have added some warning filtering in __init__

**Comment by albireox** [comment]
This seems to have been fixed for PY 3.4-3.6.
